app/Taxonomy/Department.php

```diff
@@ -43,7 +43,7 @@ public function init() {
 		$this->has_admin_menu = true;
 
 		$this->post_types = array(
-			\Boilerplate\Theme\Post_Type\Example::$ID,
+			\Boilerplate\Theme\Post_Type\Employee::$ID,
```

Comment:
add full qualified name to use section. keep simple: Employee::$ID

Reply:
done